### PR TITLE
Update: changed Input Component and 1 card component

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Changed 1 Card component and Input Component. Please the Product Card and Input Component for changes.

### CODE REVIEW

File: components/Cards/card.css [hunk 1]
```diff
@@ -277,129 +199,109 @@ span{
...
-    }
-    .horizontal-card .card-header{
-        max-height: 15em;
-        max-width: 15em;
-    }
+.product-card .product-content{
+    padding:10px 10px 5px 10px;
+    background-color: var(--clr-white);
+    max-height: 10rem;
+    overflow: hidden;
```
Comment (ApurvaSawant11) [inline components/Cards/card.css:269]: Increase the `max-height` by 0.5rem or 1 rem. The product price is cutting off a bit on full screen.
image

File: components/Cards/card-media.css [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    .horizontal-card .card-description p {
+      font-size: 1.25em;
+    }
+  
+    .horizontal-card .card-footer .btn {
+      font-size: 1.25em;
+      height: 2.5em;
+      max-width: 8em;
+      padding-left: 1em;
+      padding-right: 1em;
```
Comment (ApurvaSawant11) [inline components/Cards/card-media.css]: You can use `padding: 0 1rem`

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

Saw that you are using `em` and `rem` interchangeably.  Know the difference and use accordingly

File: components/Cards/card-media.css (hunk 2 above)
Comment (ShekharDhangar) [reply components/Cards/card-media.css]: made the changes

File: components/Cards/card.css (hunk 1 above)
Comment (ShekharDhangar) [reply components/Cards/card.css:269]: added padding

**Review by ArjunJunna — APPROVED** [review]
Nice use of media queries. Everything looks good to be merged.

**Pull request merged**